json_merger/list_unify.py

```diff
@@ -110,5 +118,10 @@ def unify(self):
             removed = self.head_stats.not_in_result_not_root_match
             conflicts.extend([Conflict(ConflictType.ADD_BACK_TO_HEAD, (), r)
                               for r in removed])
+        if self.raise_on_new_update:
+            positions_of_updates = [x[2] for x in self.unified]
+            for el in self.update:
+                if el not in self.head:
+                    conflicts.extend([Conflict(ConflictType.REMOVE_FIELD, (positions_of_updates.index(el),), el)])
```

Comment:
This can be written more compactly as
```suggestion
            conflicts.extend([Conflict(ConflictType.REMOVE_FIELD, (positions_of_updates.index(el),), el) for il in self.update if el not in self.head])
```
but I have a doubt, this looks like it just compares the two sides for equality, but the comparator should be used.

Reply:
:+1: